### PR TITLE
migrate title from controller to component

### PR DESCRIPTION
**Pull request by JonasMa**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
 <project version="4">
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectPlainTextFileTypeManager">
-    <file url="file://$PROJECT_DIR$/../scenarioo.github.io/_includes/css/main.css" />
-  </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="13" project-jdk-type="JavaSDK">
```
Comment (StuckiSimon) [inline .idea/misc.xml:7]: Have you changed the settings by accident or on purpose?

File: scenarioo-client/app/shared/navigation/title.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import * as angular from 'angular';
+import {ConfigurationService} from '../../services/configuration.service';
+import {Title} from '@angular/platform-browser';
+import {Component, Directive} from '@angular/core';
+import {downgradeComponent} from '@angular/upgrade/static';
+
+@Component({
+    selector: 'app-title',
+    template: '',
+})
```
Comment (StuckiSimon) [inline scenarioo-client/app/shared/navigation/title.component.ts:10]: Have you considered using a service? (Since there is no actual HTML involved)

**Review by StuckiSimon — COMMENTED** [review]
Looks solid, but maybe using a service would serve the purpose? Is there any benefit of using a component / directive here?

File: .idea/misc.xml (hunk 1 above)
Comment (JonasMa) [reply .idea/misc.xml:7]: accident

File: scenarioo-client/app/shared/navigation/title.component.ts (hunk 2 above)
Comment (JonasMa) [reply scenarioo-client/app/shared/navigation/title.component.ts:10]: don't you like having it on the index.html?

**Comment by JonasMa** [comment]
It would make sense to inject the Angular title service in the navigation.controller.ts but unfortunately this is still in AngularJS. When this file is migrated it's the right time (and easiest) to implement this change. Currently this wouldn't introduce any benefits

**Pull request closed without merging**